udata/templates/mail/badge_added_association.html

```diff
@@ -6,14 +6,15 @@
     _('%(user)s has identified your organization "%(name)s" as an association',
     user=(
         '<a href="'|safe
-        + url_for('api.user', user=badge.created_by, _external=True)
+        + url_for('api.user', user=badge.created_by, _external=True, **extras)
         + '">'|safe
         + badge.created_by.fullname
         + '</a>'|safe
     ),
     name=(
         '<a href="'|safe
         + organization.external_url
+        + ('?mtm_campaign=' + extras.get('mtm_campaign')) if extras.get('mtm_campaign') else ''
```

Comment:
I think some html links get broken on some mails

Reply:
What do you mean? Do you have an example I could try?
I'll spend some time soon to try setting up the actual sending of emails locally, to make sure the emails end up looking correct in my proper inbox.